pkgs/applications/audio/ardour/ardour3.nix

```diff
@@ -47,17 +53,12 @@ stdenv.mkDerivation rec {
     sed -e 's|^#!/usr/bin/env.*$|#!${perl}/bin/perl|g' -i tools/*.pl
```

Comment:
Also a note that when you use `patchShebangs` you need to include the interpreter as a `nativeBuildInput` or `buildInput`